.github/workflows/linux.yml

```diff
@@ -57,6 +57,14 @@ jobs:
       matrix: ${{fromJson(needs.matrix_prep.outputs.matrix)}}
 
     steps:
+      - id: duplicate_check
+        uses: fkirc/skip-duplicate-actions@v5
+        with:
+          # All of these options are optional, so you can remove them if you are happy with the defaults
+          skip_after_successful_duplicate: 'true'
+          paths: '["pyccel/**/*.py", "tests/**/*.py"]'
+          paths_ignore: '["pyccel/version.py"]'
```

Comment:
Is this the recommended way to use  `skip-duplicate-actions`? Using it as a job step and adding an `if` condition to every subsequent step looks a bit cumbersome to me. Maybe one can run it as an independent job and have the other jobs depend on it?

Reply:
We have the choice, but if you run it as a separate job then you can't make that job a requirement for the PR. Note that I am skipping all the steps which run the test, but not the steps which report the result